core/trashcan.js

```diff
@@ -265,19 +265,21 @@ Blockly.Trashcan.prototype.createDom = function() {
 
 /**
  * Initialize the trash can.
- * @param {number} bottom Distance from workspace bottom to bottom of trashcan.
- * @return {number} Distance from workspace bottom to the top of trashcan.
+ * @param {number} verticalSpacing Vertical distance workspace edge to the same
```

Comment:
Nit: "Vertical distance *from*"

Nit, for all such comments: indent the continuing line by +4 spaces.